salt/orch/update.sls

```diff
@@ -434,6 +434,7 @@ remove-caasp-fqdn-grain:
     - require:
       - admin-wait-for-services
 
+{%- if salt.saltutil.runner('mine.get', tgt=is_master_tgt + ' and G@update_in_progress:true', fun='network.interfaces', tgt_type='compound')|length > 0 %}
```

Comment:
It would be better to use the `is_updateable_master_tgt` variable previously defined, so

```Jinja
{%- if salt.saltutil.runner('mine.get', tgt=is_updateable_master_tgt, fun='network.interfaces', tgt_type='compound')|length > 0 %}
```

I would even add a `updateable_master` masters affter the `all_masters` definition.

```Jinja
{%- set updateable_masters = salt.saltutil.runner('mine.get', tgt=is_updateablee_master_tgt, fun='network.interfaces', tgt_type='compound').keys() %}
...
{%- if updateable_masters|length > 0 %}
...
{% endif %}
```

Reply:
I've just realized the `masters` variable holds that info. This would be enough:

```Jinja
{%- if masters|length > 0 %}
...
{% endif %}
```

Reply:
No, it's not the same. `is_updateable_master_tgt` has `G@tx_update_reboot_needed:true and G@roles:kube-master` whereas we want `G@update_in_progress:true and G@roles:kube-master`.

Reply:
We cannot use `masters` because that list is fetched using `is_updateable_master_tgt` that doesn't really match what we want on this step.

Reply:
You are right, but then I think we will have a proble, because the Jinja code is evaluated once at beggining, so I think you cannot use a grain that is set in the orchestration.

Reply:
Intelligently solved with a grain target. I think we are done.